### PR TITLE
Update Elasticsearch Connector docs for Search Guard

### PR DESCRIPTION
**Pull request by TempleZhou**

### CODE REVIEW

File: presto-docs/src/main/sphinx/connector/elasticsearch.rst [hunk 1]
```diff
@@ -103,7 +103,7 @@ This property defines the timeout value for all Elasticsearch requests.
 This property is optional; the default is ``10s``.
 
 ``elasticsearch.max-request-retries``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (nezihyigitbasi) [inline presto-docs/src/main/sphinx/connector/elasticsearch.rst]: Please move this change to a separate commit.

File: presto-docs/src/main/sphinx/connector/elasticsearch.rst [hunk 2]
```diff
@@ -116,6 +116,99 @@ Use exponential backoff starting at 1s up to the value specified by this configu
 
 This property is optional; the default is ``10s``.
 
+Search Guard Authentication
+---------------------------
+
+The Elasticsearch connector provides additional security options to support Elasticsearch clusters that have been configured to use Search Guard.
+
+You can choose certificate format by setting the ``searchguard.ssl.transport.certificate_format`` in the Elasticsearch catalog properties file. This property must be one of the following values:
```
Comment (nezihyigitbasi) [inline presto-docs/src/main/sphinx/connector/elasticsearch.rst]:
```
You can configure the certificate format by setting the ``searchguard.ssl.transport.certificate_format`` config property ...
```

Comment (nezihyigitbasi) [reply presto-docs/src/main/sphinx/connector/elasticsearch.rst]: `The allowed values for this configuration are:`

File: presto-docs/src/main/sphinx/connector/elasticsearch.rst [hunk 3]
```diff
@@ -116,6 +116,99 @@ Use exponential backoff starting at 1s up to the value specified by this configu
...
+
+========================== ========================================================
+Property Value	           Description
+========================== ========================================================
+``NONE`` (default value)   Do not use Search Guard Authentication
```
Comment (nezihyigitbasi) [inline presto-docs/src/main/sphinx/connector/elasticsearch.rst]: `(default)`

File: presto-docs/src/main/sphinx/connector/elasticsearch.rst [hunk 4]
```diff
@@ -116,6 +116,99 @@ Use exponential backoff starting at 1s up to the value specified by this configu
...
+``PEM``                    Use X.509 PEM certificates and PKCS #8 keys 
+``JKS``                    Use Keystore and Truststore files
```
Comment (nezihyigitbasi) [inline presto-docs/src/main/sphinx/connector/elasticsearch.rst]: Please put `.` at the end of the descriptions as we do in other parts of the doc.

File: presto-docs/src/main/sphinx/connector/elasticsearch.rst [hunk 5]
```diff
@@ -116,6 +116,99 @@ Use exponential backoff starting at 1s up to the value specified by this configu
...
+========================== ========================================================
+
+If you use X.509 PEM certificates and PKCS #8 keys, you must set the following properties:
```
Comment (nezihyigitbasi) [inline presto-docs/src/main/sphinx/connector/elasticsearch.rst]: Instead of `you must set` I think we can say `the following properties must be set`. Please update all instances in this PR.

Comment (TempleZhou) [reply presto-docs/src/main/sphinx/connector/elasticsearch.rst]: Fixed.

**Comment by nezihyigitbasi** [comment]
Merged, thanks!
